www/js/control/ProfileSettings.jsx

```diff
@@ -150,54 +255,206 @@ const ProfileSettings = () => {
+            if(prevTime + 2000 < currTime ){
+                toggleLowAccuracy();
+                setToggleTime(new Date());
+            }
+        }
+        else {
+            toggleLowAccuracy();
+            setToggleTime(new Date());
+        }
+    }
+
+    async function toggleLowAccuracy() {
         ControlCollectionHelper.toggleLowAccuracy();
         refreshCollectSettings();
     }
 
     const shareQR = function() {
-        var prepopulateQRMessage = {};  
-        var qrAddress = "emission://login_token?token="+settings?.auth?.opcode;
-        prepopulateQRMessage.files = [qrAddress];
-        prepopulateQRMessage.url = settings.auth.opcode;
-
-        window.plugins.socialsharing.shareWithOptions(prepopulateQRMessage, function(result) {
-            console.log("Share completed? " + result.completed); // On Android apps mostly return false even while it's true
-            console.log("Shared to app: " + result.app); // On Android result.app is currently empty. On iOS it's empty when sharing is cancelled (result.completed=false)
-        }, function(msg) {
-            console.log("Sharing failed with message: " + msg);
-        });
+       /*code adapted from demo of react-qr-code*/
+        const svg = document.querySelector(".qr-code");
+        const svgData = new XMLSerializer().serializeToString(svg);
+        const img = new Image();
+
+        img.onload = () => {
+            const canvas = document.createElement("canvas");
+            const ctx = canvas.getContext("2d");
+            canvas.width = img.width;
+            canvas.height = img.height;
+            ctx.drawImage(img, 0, 0);
+            const pngFile = canvas.toDataURL("image/png");
+                
+            var prepopulateQRMessage = {}; 
+            prepopulateQRMessage.files = [pngFile];
+            prepopulateQRMessage.url = authSettings.opcode;
+            prepopulateQRMessage.message = authSettings.opcode; //text saved to files with image!
+                
+            window.plugins.socialsharing.shareWithOptions(prepopulateQRMessage, function(result) {
+                console.log("Share completed? " + result.completed); // On Android apps mostly return false even while it's true
+                console.log("Shared to app: " + result.app); // On Android result.app is currently empty. On iOS it's empty when sharing is cancelled (result.completed=false)
+            }, function(msg) {
+                console.log("Sharing failed with message: " + msg);
+            });
+        }
+        img.src =  `data:image/svg+xml;base64,${btoa(svgData)}`;
     }
 
     const viewQRCode = function(e) {
         setOpCodeVis(true);
     }
+    
+    const clearNotifications = function() {
+        window.cordova.plugins.notification.local.clearAll();
+    }
 
-    var prepopulateMessage = {
-        message: t('general-settings.share-message'),
-        subject: t('general-settings.share-subject'),
-        url: t('general-settings.share-url')
```

Comment:
wrt https://github.com/e-mission/e-mission-phone/pull/1009#issuecomment-1686783716

> While I'm in settings, should I update this share to reflect NREL OPENPath and an nrel-hosted download site, or is this information still acurate?

we decided to just remove that functionality, correct?

Reply:
correct, we removed "sharing the app" from the profile mostly because we're not doing our own recruiting anymore